src/sidebar/services/load-annotations.ts

```diff
@@ -191,7 +197,7 @@ export class LoadAnnotationsService {
       //    fetch the top-level annotation
       if (isReply(annotation)) {
         annotation = await this._api.annotation.get({
-          id: /** @type {string[]} */ (annotation.references)[0],
+          id: annotation.references![0],
```

Comment:
The `isReply` function involves checking to see if there are `references`, so I think it's probably safe to assume the presence here.